10-open.md

```diff
@@ -2,7 +2,7 @@
 layout: page
 title: Version Control with Git
 subtitle: Open Science
-minutes: 5
+minutes: 5-10
```

Comment:
You can put 10 minutes for this: 5 to explain the lesson and 5 to do a challenge.